### ISSUE TITLE
sxnippet is NOT installed.

### ISSUE THREAD

**Issue by demiscuzz** [MISP/MISP#6902]
I am trying to do a custom install of misp on ubunut 20 using the guide  @ https://github.com/MISP/MISP/tree/2.4/INSTALL   I have followed the guide exactly and i am getting and error when i run ./INSTALL.tpl.sh 
xsnippet is NOT installed. Clone the repository below and copy the xsnippet shell script somehwere in your $PATH
git clone https://github.com/SteveClement/xsnippet.git

# echo $PATH
/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin:~/bin:~/bin:/root/xsnippet/MISP/INSTALL/xsnippet/xsnippet.sh:~/bin

Please tell me what I am missing.  Super big thank you and sorry if this is a dumb issue.

**Comment by chrisinmtown** [comment]
Um, did you mean sxnippet or xsnippet? And maybe this is a dumb question in response, but after you cloned Steve Clement's repo, where did you copy the xsnippet shell script?  What is the output from the shell command `which xsnippet` ?

**Comment by demiscuzz** [comment]
Just started a fresh vm same issue.  Yes I did mean xsnippet. As for copying it I followed the guide: "https://github.com/MISP/MISP/tree/2.4/INSTALL"

"You will need xsnippet that extracts bits of shell code from the .md files:

mkdir -p ~/bin
git clone https://github.com/SteveClement/xsnippet.git
cd xsnippet; cp xsnippet ~/bin/
export PATH="$PATH:~/bin" # By now you are aware that this needs to be in your $PATH, aren't you. #PAAF
You need rhash too:

sudo apt install rhash
Now you are ready.

To test if you are really ready, do the following:

git clone https://github.com/MISP/MISP.git
cd MISP/INSTALL ; ./INSTALL.tpl.sh "
 
So its in the ~/bin 
I have also copied it to /usr/local/sbin

If i use 
"which xsnippet" 
it returns nothing.

**Comment by demiscuzz** [comment]
Thank you for your reply. You prompted me to learn what the command 'which' does and this helped me find the issue. 
I followed the guide 'https://github.com/MISP/MISP/tree/2.4/INSTALL' 
In the guide it says to copy the xsnippet into ~/bin . This was in my $PATH but for some reason not working. I copied xnsippet into /usr/local/sbin and then which xsnippet showed the path and now the install works. Thank you so much.
